database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,9 +15,8 @@ public function up(): void
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
-            $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
+            $table->string('mail')->unique();
```

Comment:
We usually use email for this field, mail is used more for when we send out the mails